blockchain/state_manager/src/lib.rs

```diff
@@ -388,7 +387,8 @@ where
         let receipts = vm.apply_block_messages(messages, epoch, callback)?;
 
         // Construct receipt root from receipts
-        let receipt_root = Amt::new_from_iter(self.blockstore(), receipts)?;
+        let receipt_root = Amt::new_from_iter(self.blockstore(), receipts)
+            .map_err(|e| anyhow::anyhow!("{}", e))?;
```

Comment:
`e.into()` should work as well, no? a bit more succinct

Reply:
Not sure how to do.

Reply:
`.map_err(|e| e.into())?;` doesn't work?

Reply:
`.map_err(|e| e.into())?` is the same as just `?`, right?

Reply:
No, not out of the box.

Reply:
worked on the code I was browsing, carry on then